docs/environments/unarchived-environment.md

```diff
@@ -0,0 +1,17 @@
+---
+draft: true
+title: Unarchive an Environment
+intro: Some environments may need to be unarchived to re-run tests or manage a service included in the environment. Unarchive an Environment to view its linked resources.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, access the archived environment list
+    > Follow the [Find an Archived Environment](/docs/environments/find-archived-environment) guide
+1. Find the `Environment` you want to unarchive and click `UNARCHIVE`
```

Comment:
This last step should not be needed, as the linked topic already guides the reader to `Find an Archived Environment`, a specific one.

Reply:
Post updated, now the `Unarchive` flow it's proposed to be accessed at the `Environment` resource list page, on the submenu dropdown